Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2732,14 +2880,33 @@ extension BrowserViewController: KeyboardHelperDelegate {
             self.findInPageContainer.layoutIfNeeded()
             self.snackBars.layoutIfNeeded()
         }
+
+        if let webView = tabManager.selectedTab?.webView {
+            webView.evaluateJavaScript("__firefox__.isActiveElementSearchField()") {[weak self] (result, _) in
```

Comment:
Nit: no need for `[weak self]`. This block is immediately evaluated and not retained.